acceptance/tests/runs_external_facts_once.rb

```diff
@@ -47,13 +43,14 @@
   on(agent, "chmod +x '#{ext_fact}'")
 
   step "Agent #{agent}: ensure it only executes once"
-  on(agent, facter) do
-    lines = stderr.split('\n')
+  on(agent, facter('test'))
```

Comment:
Should we leave this just `facter` to ensure that during all fact evaluation, the external fact is only executed once?